homeassistant/components/twinkly/light.py

```diff
@@ -304,6 +303,9 @@ async def async_update(self) -> None:
             if LightEntityFeature.EFFECT & self.supported_features:
                 await self.async_update_movies()
                 await self.async_update_current_movie()
+                await self.async_update_current_color()
```

Comment:
Are these three awaits required to happen in order? If not, you could execute them simultaneously:

```python
await asyncio.gather(
    *[
        self.async_update_movies(),
        self.async_update_current_movie(),
        self.async_update_current_color(),
    ]
)
```

Reply:
Thanks.